src/Servers/IIS/IntegrationTesting.IIS/src/IISDeployer.cs

```diff
@@ -24,12 +24,13 @@ public class IISDeployer : IISDeployerBase
         private const string DetailedErrorsEnvironmentVariable = "ASPNETCORE_DETAILEDERRORS";
 
         private static readonly TimeSpan _timeout = TimeSpan.FromSeconds(60);
-        private static readonly TimeSpan _retryDelay = TimeSpan.FromMilliseconds(200);
+        private static readonly TimeSpan _retryDelay = TimeSpan.FromMilliseconds(500);
```

Comment:
Why this?

Reply:
Trying to make logs less spammy. I'll make it quadratic.